src/common/resources/contacts.js

```diff
@@ -1,10 +1,15 @@
 import { CONTACTS_API } from '../constants';
 
-// eslint-disable-next-line import/prefer-default-export
 export const contactResource = {
   type: 'okapi',
   path: (queryParams, pathComponents) => {
     if (pathComponents.id) return `${CONTACTS_API}/${pathComponents.id}`;
     return undefined;
   },
 };
+
+export const baseContactsResource = {
```

Comment:
what if to call it `contactsResource` ?

Reply:
`contactsResource` can be different so I called it `base`